scripts/docker_entrypoint.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -e
+
+randomhex() {
+	local size="$1"
+	if [ -z "${size}" ]; then
+		size=32
+	fi
+	local val=$(hexdump -e '4/4 "%08x"' -n${size} /dev/random)
+	echo ${val}
+}
+
+if [ "$NEWCERT" = "1" -o ! -e /srv/cert.pem ]; then
+	echo "Creating new self signed TLS certificate ..."
+	rm -f /srv/privkey.pem
+	rm -f /srv/cert.pem
+	openssl ecparam -genkey -name secp384r1 -out /srv/privkey.pem
+	openssl req -new -x509 -key /srv/privkey.pem \
+				-out /srv/cert.pem -days 3650 \
+				-subj /CN=spreed-webrtc \
+				-config /etc/ssl/openssl.cnf \
+				-sha256 -extensions v3_req
+
+fi
+echo "TLS certificate:"
+openssl x509 -in /srv/cert.pem -text
+
+if [ "$NEWSECRETS" = "1" -o ! -e /srv/secrets.conf ]; then
```

Comment:
Nit: use `-s` instead of `-e` to also generate secrets if file exists but is empty.